### DISCUSSION TITLE
Avalonia Webview overlaying

### DISCUSSION THREAD

**Discussion by geelapidus** [AvaloniaUI/Avalonia#17697] [Q&A]
in ver 11 how to overlay webview by a user control ?  I tried to wrap it into Popup and/or canvas  without success

**Comment by msneijders** [comment]
When embedding native controls in your avalonia view there currently is no (easy) built-in way to get avalonia controls on top of it. You probably need to jump to quite some technical hoops to get that working.

**Comment by StefanKoell** [comment]
@geelapidus I tried Popups with NativeControlHost a while back and this approach worked for me. I can't find my prototype/PoC anymore but maybe you can share a sample app using Popups and let us have a look.
